docs/source/development-guidelines.rst

```diff
@@ -0,0 +1,205 @@
+Zipline Development Guidelines
+==============================
+This page is intended for developers of Zipline, people who want to contribute to the Zipline codebase or documentation, or people who want to install from source and make local changes to their copy of Zipline.
+
+All contributions, bug reports, bug fixes, documentation improvements, enhancements and ideas are welcome. We `track issues`__ on `GitHub`__ and also have a `mailing list`__ where you can ask questions.
+
+__ https://github.com/quantopian/zipline/issues
+__ https://github.com/
+__ https://groups.google.com/forum/#!forum/zipline
+
+Creating a Development Environment
+----------------------------------
+
+First, you'll need to clone Zipline by running:
+
+.. code-block:: bash
+
+   git clone git@github.com:your-github-username/zipline.git
+
+Then check out to a new branch where you can make your changes:
+
+.. code-block:: bash
+		
+   git checkout -b some-short-descriptive-name
```

Comment:
Just noticing that we're inconsistent in using "$" to denote the bash prompt in our code block.